src/components/TodoList/TodoList.js

```diff
@@ -12,53 +12,76 @@ import {
 
 function TodoList({ itemValues }) {
   const initialState = {
-    todoItem: [],
+    todoItems: [],
     filter: "all",
   };
 
   const [state, setState] = useState(initialState);
-  const { todoItem, filter } = state;
+  const { todoItems } = state;
 
   const filterTodoItems = (status) => {
     switch (status) {
       case "all":
-        return todoItem;
+        return todoItems;
       case "checked":
-        const checkedItems = todoItem.filter(
+        const checkedItems = todoItems.filter(
           (item) => item.status === "checked"
         );
         return checkedItems;
       case "crossed":
-        const crossedItems = todoItem.filter(
+        const crossedItems = todoItems.filter(
           (item) => item.status === "crossed"
         );
         return crossedItems;
     }
   };
 
   const getNextId = (todoInput) => {
-    return todoItem.length + todoInput.toString();
+    return todoItems.length + todoInput.toString();
   };
 
   const createTodoItem = (itemValue) => {
-    setState({ ...state, todoItem: [...todoItem, itemValue] });
+    setState({ ...state, todoItems: [...todoItems, itemValue] });
   };
 
   const onSubmitTodoContent = (todoInput) => {
     const newItem = {
       content: todoInput,
       status: "unchecked",
       id: getNextId(todoInput),
+      strikeThrough: false,
     };
     createTodoItem(newItem);
   };
 
   const updateTodoItemStatus = (id, checkStatus) => {
     setState((current) => {
-      const updatedItem = current.todoItem.map((item) =>
+      const updatedItemStatus = current.todoItems.map((item) =>
         item.id === id ? { ...item, status: checkStatus } : item
       );
-      return { ...current, todoItem: updatedItem };
+      return { ...current, todoItems: updatedItemStatus };
+    });
+  };
+
+  const handleUpdateStrikeThrough = (id) => {
+    setState((current) => {
+      const itemClicked = current.todoItems.find((item) => item.id === id);
+      let updatedItemStrike;
+      if (!itemClicked) return;
+      if (itemClicked.strikeThrough === true) {
+        updatedItemStrike = current.todoItems.filter((item) => item.id !== id);
```

Comment:
nên đặt là `const updatedItemStrike`, bỏ let updatedItemStrike ở trên đi

nên ưu tiên dùng const bất cứ khi nào có thể. chỉ dùng let khi không thể dùng const

https://stackoverflow.com/questions/41086633/why-most-of-the-time-should-i-use-const-instead-of-let-in-javascript